### PR TITLE
Expose disabled property in IColorCellProps

### PR DESCRIPTION
**Pull request by jessie782**
## Previous Behavior

The disabled property is not exported in the IColorCellProps interface.

## New Behavior

The disabled property is exported as part of the IColorCellProps interface to allow users to disable color cells.

## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/d145902686da811aeb90e506b829044e1fb3cf90'>d145902686da811aeb90e506b829044e1fb3cf90</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-f8187229-7071-4c6a-be6e-1ea511a021e2.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-f8187229-7071-4c6a-be6e-1ea511a021e2.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 8 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28015&commitId=757f7a4b366c448938001929c4c3a9b779043220&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1039 screenshots unchanged, 0 screenshots with different dimensions and 6 screenshots with visible difference.


  <details><summary><b>unknown</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchColorPicker.Disabled.Focused.chromium.png|482| Changed|
|SwatchColorPicker.Disabled.Hovered.chromium.png|440| Changed|
|SwatchColorPicker.Disabled.Mousedown.chromium.png|437| Changed|
|SwatchColorPicker.Disabled.Selected and Focused.chromium.png|437| Changed|
|SwatchColorPicker.Disabled.Selected and Hovered.chromium.png|437| Changed|
|SwatchColorPicker.Disabled.default.chromium.png|479| Changed|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/BaseButton-mount_base.html">628</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/BaseButton-mount.html">636</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Breadcrumb-mount_base.html">1673</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Breadcrumb-mount.html">1684</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Checkbox-mount_base.html">1708</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Checkbox-mount.html">1696</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/CheckboxBase-mount_base.html">1495</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/CheckboxBase-mount.html">1494</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/ChoiceGroup-mount_base.html">2987</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/ChoiceGroup-mount.html">2948</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/ComboBox-mount_base.html">655</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/ComboBox-mount.html">653</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/CommandBar-mount_base.html">6326</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/CommandBar-mount.html">6308</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/ContextualMenu-mount_base.html">13474</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/ContextualMenu-mount.html">13689</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/DefaultButton-mount_base.html">757</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/DefaultButton-mount.html">747</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/DetailsRow-mount_base.html">2193</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/DetailsRow-mount.html">2221</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/DetailsRowFast-mount_base.html">2151</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/DetailsRowFast-mount.html">2199</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/DetailsRowNoStyles-mount_base.html">2029</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/DetailsRowNoStyles-mount.html">2016</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Dialog-mount_base.html">2831</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Dialog-mount.html">2782</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/DocumentCardTitle-mount_base.html">230</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/DocumentCardTitle-mount.html">226</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Dropdown-mount_base.html">1982</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Dropdown-mount.html">2025</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/FocusTrapZone-mount_base.html">1102</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/FocusTrapZone-mount.html">1149</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/FocusZone-mount_base.html">1053</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/FocusZone-mount.html">1031</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/GroupedList-mount_base.html">41368</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/GroupedList-mount.html">41613</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/GroupedList-virtual-rerender_base.html">17988</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/GroupedList-virtual-rerender.html">20180</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">50895</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/GroupedList-virtual-rerender-with-unmount.html">50849</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/GroupedListV2-mount_base.html">216</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/GroupedListV2-mount.html">237</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/GroupedListV2-virtual-rerender_base.html">207</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/GroupedListV2-virtual-rerender.html">215</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">229</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">226</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/IconButton-mount_base.html">1087</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/IconButton-mount.html">1081</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Label-mount_base.html">337</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Label-mount.html">333</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Layer-mount_base.html">2714</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Layer-mount.html">2717</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Link-mount_base.html">386</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Link-mount.html">401</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/MenuButton-mount_base.html">937</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/MenuButton-mount.html">958</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/MessageBar-mount_base.html">21459</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/MessageBar-mount.html">21436</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Nav-mount_base.html">1955</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Nav-mount.html">1955</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/OverflowSet-mount_base.html">769</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/OverflowSet-mount.html">775</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Panel-mount_base.html">1744</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Panel-mount.html">1750</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Persona-mount_base.html">777</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Persona-mount.html">754</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Pivot-mount_base.html">877</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Pivot-mount.html">862</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/PrimaryButton-mount_base.html">855</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/PrimaryButton-mount.html">859</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Rating-mount_base.html">4661</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Rating-mount.html">4692</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/SearchBox-mount_base.html">915</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/SearchBox-mount.html">908</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Shimmer-mount_base.html">1846</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Shimmer-mount.html">1902</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Slider-mount_base.html">1377</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Slider-mount.html">1365</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/SpinButton-mount_base.html">2876</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/SpinButton-mount.html">2898</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Spinner-mount_base.html">393</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Spinner-mount.html">379</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/SplitButton-mount_base.html">1822</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/SplitButton-mount.html">1851</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Stack-mount_base.html">427</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Stack-mount.html">401</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/StackWithIntrinsicChildren-mount_base.html">878</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/StackWithIntrinsicChildren-mount.html">877</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/StackWithTextChildren-mount_base.html">2690</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/StackWithTextChildren-mount.html">2691</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/SwatchColorPicker-mount_base.html">6092</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/SwatchColorPicker-mount.html">6174</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/TagPicker-mount_base.html">1460</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/TagPicker-mount.html">1463</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Text-mount_base.html">377</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Text-mount.html">384</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/TextField-mount_base.html">935</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/TextField-mount.html">961</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/ThemeProvider-mount_base.html">830</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/ThemeProvider-mount.html">846</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/ThemeProvider-virtual-rerender_base.html">587</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/ThemeProvider-virtual-rerender.html">587</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1288</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1279</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Toggle-mount_base.html">608</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/Toggle-mount.html">621</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/buttonNative-mount_base.html">195</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28015/perf-test/buttonNative-mount.html">189</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by jessie782** [comment]
> @jessie782 please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree company="Microsoft"

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Review by khmakoto — APPROVED** [review]

**Pull request merged**